src/tfr_navigation/src/navigation_client.cpp

```diff
@@ -0,0 +1,69 @@
+#include <navigation_client.h>
+        
+/**
+ *  default constructor, brings up a server to listen to a specific action
+ * */
+NavigationClient::NavigationClient(std::string action_name):client{action_name, true}
+{
+    client.waitForServer();
+}
+
+
+/**
+ * Nonblocking call to Navigate to an arbirtrary defined place as defined in the Navigation
+ * Goal constants header.
+ *
+ * action - The byte code corresponding to a location in the world map
+ * */
+void NavigationClient::navigate(uint8_t location)
+{
+    tfr_msgs::NavigationGoal goal;
```

Comment:
Is a `tfr_msgs::NavigationGoal` a `uint8_t`? If so, why not just have the parameter be named "goal" instead?

Reply:
See above comment I want to replace this with a strongly typed c++ enum. on the next pull request.